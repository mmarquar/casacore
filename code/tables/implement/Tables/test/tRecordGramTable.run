#!/bin/sh
#-----------------------------------------------------------------------------
# Script to test the RecordGram class for a table
#=============================================================================

tRecordGramTable "ab"
tRecordGramTable "ac"
tRecordGramTable "ad"
tRecordGramTable "ae"
tRecordGramTable "af"
tRecordGramTable "ag"
tRecordGramTable "ab>4"
tRecordGramTable "ab+ac"
