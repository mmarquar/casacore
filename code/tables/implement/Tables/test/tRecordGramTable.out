ab: [0, 1, 2, 3, 4, 5, 6, 7, 8, 9]
ac: [1, 2, 3, 4, 5, 6, 7, 8, 9, 10]
ad: [2, 3, 4, 5, 6, 7, 8, 9, 10, 11]
ae: [3, 4, 5, 6, 7, 8, 9, 10, 11, 12]
af: [V0, V1, V2, V3, V4, V5, V6, V7, V8, V9]
ag: [(2,0), (3,0), (4,0), (5,0), (6,0), (7,0), (8,0), (9,0), (10,0), (11,0)]
ab>4: [0, 0, 0, 0, 0, 1, 1, 1, 1, 1]
ab+ac: [1, 3, 5, 7, 9, 11, 13, 15, 17, 19]
